### PR TITLE
New single balance ReadPool

### PR DESCRIPTION
**Pull request by iamrz1**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/1380
- blobber:https://github.com/0chain/blobber/pull/723
- zbox-cli: https://github.com/0chain/zboxcli/pull/256
- system_test: https://github.com/0chain/system_test/pull/297

### CODE REVIEW

File: zboxcore/sdk/sdk.go [hunk 1]
```diff
@@ -227,45 +230,28 @@ func GetReadPoolInfo(clientID string) (info *ReadPool, err error) {
 }
 
 // ReadPoolLock locks given number of tokes for given duration in read pool.
-func ReadPoolLock(tokens, fee int64, isOwner bool) (hash string, nonce int64, err error) {
+func ReadPoolLock(tokens, fee int64) (hash string, nonce int64, err error) {
 	if !sdkInitialized {
 		return "", 0, sdkNotInitialized
 	}
 
-	type lockRequest struct {
```
Comment (dabasov) [inline zboxcore/sdk/sdk.go:235]: will it work without request?

File: zboxcore/sdk/sdk.go [hunk 2]
```diff
@@ -227,45 +230,28 @@ func GetReadPoolInfo(clientID string) (info *ReadPool, err error) {
...
 }
 
 // ReadPoolUnlock unlocks tokens in expired read pool
-func ReadPoolUnlock(fee int64, isOwner bool) (hash string, nonce int64, err error) {
+func ReadPoolUnlock(fee int64) (hash string, nonce int64, err error) {
 	if !sdkInitialized {
 		return "", 0, sdkNotInitialized
 	}
 
-	type unlockRequest struct {
```
Comment (dabasov) [inline zboxcore/sdk/sdk.go:258]: will it work without request?

File: zboxcore/sdk/sdk.go (hunk 1 above)
Comment (iamrz1) [reply zboxcore/sdk/sdk.go:235]: yes, tested both manually and with custom system-test.

File: zboxcore/sdk/sdk.go (hunk 2 above)
Comment (iamrz1) [reply zboxcore/sdk/sdk.go:258]: yes, tested both manually and with custom system-test.

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | master  |
| gosdk  | single-balance-readpool  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**